ios/MaasCore/Sources/MaasCore/Core.xcframework/Info.plist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As discussed yesterday, let's not commit the xcframework to git.
1. It's massive.
1. It duplicates the Kotlin source code which is the actual "source of truth".
1. There is no guarantee that the committed xcframework is up-to-date with the committed Kotlin source code.
1. It would result in hard-to-read PR diffs (such as this one 😉).

Reply:
Removed. 🔥